### DISCUSSION TITLE
Invalid cells and stencils using Version 4.1

### DISCUSSION THREAD

**Discussion by Fracturist** [WENO-OF/WENOEXT#46] [Q&A]
Hello,

I have recently noticed appearance of extra invalid cells and stencils in my own test case when using Version 4.1.

![image](https://user-images.githubusercontent.com/103815630/176339304-c7e76fca-fabc-4981-8cb9-5368174fa682.png)

Then I checked the log file stored previously for same case using Version 4.0. It seems that everything is fine.

![image2](https://user-images.githubusercontent.com/103815630/176340699-87ce16e2-eb94-428b-af72-92cf3ee60969.png)

So I did a rerun for tutorial case using Version 4.1. Serial Cavity case is OK but parallel Cavity case showed an extra invalid cell. Moreover, all cells in Wedge case seem invalid. I am pretty sure that these case was fine with Version 4.0.

![Selection_004](https://user-images.githubusercontent.com/103815630/176341985-62e374e2-03b5-40b6-90a5-cee8365ecb2b.png)

The log fie for tutorial cases is here
[log.txt](https://github.com/WENO-OF/WENOEXT/files/9006906/log.txt)

Can someone give me a hint as to how these happened? Is it due to my compilation or some feature related to new version?

**Comment by JanGaertner** [comment]
Hello Facturist,
I have moved this discussion to issues. I have replied there as well :)
